functionary/core/utils/tasking.py

```diff
@@ -250,6 +263,22 @@ def start_task(task: Task) -> None:
     elif tasked_type_class is Workflow:
         _start_workflow_task(task)
     else:
-        raise InvalidContentObject(
-            f"Handling for content type {tasked_type_class} is undefined"
+        message = f"Handling for content type {tasked_type_class} is undefined"
+        task_errored(task, message, None)
+        raise InvalidContentObject(message)
+
+
+def task_errored(task, message, error=None):
```

Comment:
name this something more action-y, inline with `start_task` and `publish_task`.

Reply:
done